src/const.ts

```diff
@@ -0,0 +1,15 @@
+export const cities: string[] = [
```

Comment:
это константа, оформляем синтаксисом
```suggestion
export const CITIES: string[] = [
```